pkg/interactor/save_data_test.go

```diff
@@ -0,0 +1,113 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	thingentities "github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+var date = time.Date(2020, time.April, 15, 0, 0, 0, 0, time.UTC)
+
+type SaveDataTestCase struct {
+	name               string
+	authorization      string
+	deviceID           string
+	payload            []entities.Payload
+	data               entities.Data
+	expectedData       *entities.Data
+	expectedDataError  error
+	expectedThing      []*thingentities.Thing
+	expectedThingError error
```

Comment:
I think one valid solution to avoid this property's mess would be moving them to each mock as we did [here](https://github.com/CESARBR/knot-babeltower/blob/master/pkg/thing/mocks/thing_proxy.go#L13).

Reply:
Done!